src/AddBookForm.js

```diff
@@ -5,51 +5,67 @@ import useForm from 'react-hook-form';
 import { TextField, Button, Grid, createStyles, makeStyles } from '@material-ui/core';
 
 const mutation = gql`
-  mutation AddBook($title: String!, $author: String!) {
-    addBook(title: $title, author: $author) {
+  mutation AddBook($title: String!, $author: String!, $numPages: Int!) {
+    addBook(title: $title, author: $author, numPages: $numPages) {
       id
       title
       author
+      numPages
     }
   }
 `;
 
 const useStyle = makeStyles((theme) =>
   createStyles({
-    item: {
-      margin: theme.spacing(1),
+    root: {
+      width: '60%'
```

Comment:
Why 60%?
How does this behave on narrow screens such as mobile?

Reply:
You right.
I remove the width:60% and add a Container of material ui.